pkgs/development/tools/misc/gede/build.patch

```diff
@@ -0,0 +1,35 @@
+--- a/build.py	2017-01-16 21:12:43.000000000 +0100
++++ b/build.py	2017-02-26 22:03:11.394625315 +0100
+@@ -71,7 +71,7 @@
+         if do_build:
+             if not os.path.exists("Makefile"):
+                 print("Generating makefile")
+-                if subprocess.call(['qmake-qt4']):
++                if subprocess.call(['qmake']):
+                     exit(1)
+ 
+             print("Compiling (please wait)")
+@@ -88,7 +88,7 @@
+                 exit(1)
+             try:
+                 shutil.copyfile("gede", g_dest_path + "/bin/gede")
+-                os.chmod(g_dest_path + "/bin/gede", 0775);
++                os.chmod(g_dest_path + "/bin/gede", 775);
```

Comment:
The argument to chmod needs to be octal:
In python3 this is done by prefixing `0o`

```
 os.chmod(g_dest_path + "/bin/gede", 0o755)
```